### ISSUE TITLE
Cross databases tasks

### ISSUE THREAD

**Issue by AlexGrs** [influxdata/kapacitor#808]
Hi there. 

Is there a recommended way to run a cross database task ? Let's say I have two databases with the same measurement `page_count_status`. 

I would like to use kapacitor to compute the success / rate (ie `page_count_status success / (page_count_status success  + page_count_status failed`) for each database.

Do I need to create two tasks for this or is it a way to have one tasks for all DB ?

Thanks.

**Comment by nathanielc** [comment]
@AlexGrs If I understand right, you have two or more identical databases and you want to perform the same operation across all databases keeping the results separated? Correct?

Unfortunately there is now way to do this within a single task. The best thing you can do right now is to create a `task-template` and have the database be a var in the template. Then define a new task off the template for each database.

Essentially what you want is the ability to add the database name as a tag on the data so you can group by it and then you could use a single task for all databases.

**Comment by nathanielc** [comment]
Currently its possible to group by the measurement name but that doesn't help in this case. We should add a mechanism for being able to add the database and retention policy to the group ID as well.

**Comment by AlexGrs** [comment]
Yep, this feature would be highly appreciated. I will check and try the template solution for the moment.
